apps/api/src/domain/models/playlist/errors/cannot-add-duplicate-item-to-playlist.error.ts

```diff
@@ -0,0 +1,18 @@
+import { InternalError } from '../../../../lib/errors/InternalError';
+import formatAggregateCompositeIdentifier from '../../../../view-models/presentation/formatAggregateCompositeIdentifier';
+import { AggregateId } from '../../../types/AggregateId';
+import { AggregateType } from '../../../types/AggregateType';
+import { PlaylistItem } from '../entities/playlist-item.entity';
+
+export class CannotAddDuplicateItemToPlaylist extends InternalError {
```

Comment:
We need to add a `checkError` to the corresponding test case. This is the error you are looking for.

Reply:
done